vendor/engines/saas/lib/saas/engine.rb

```diff
@@ -17,6 +17,20 @@ class Engine < ::Rails::Engine
         })
       end
 
+      LoginController.class_eval do
+        before_action :page_job, only: [:public, :private]
+
+        :private
+
+        def page_job
+          auth_request = request.params['code'] && request.params['state']
+          if request.referer !~ /github\.com/ && !logged_in? && !auth_request
```

Comment:
Right, but this particularly code is mean to log `/login/{scope}` only at step 2. Step 5 handles the post-auth tracking of `/login/{scope}/authorized`, unless I'm missing something?